Puzzlepart/prosjektportalen365
# Added check to limit generation of project property elements

### Your checklist for this pull request


### Description

While testing Program, we noticed that the wizard will write more than 1 element to the project properties if additional templates is added. The bug can also be triggered if a user cancel and rerun the wizard (e.g. page refresh). The bug caused issues when we were working with program. 

### How to test

1. Create a new project
2. Select a template and run the wizard
3. Abort the setup (do a page refresh)
4. The wizard will now restart
5. Let the wizard completely setup the project
6. Check /lists/prosjektegenskaper
7. Only one element should be displayed

Updating the project properties, testing the sync to hub should also be tested..

### Relevant issues
Fixes #603 
Closes #603 

💔Thank you!
